firstvoices/backend/resources/characters.py

```diff
@@ -1,12 +1,24 @@
 from import_export import fields
-from import_export.widgets import ForeignKeyWidget
+from import_export.widgets import ForeignKeyWidget, ManyToManyWidget
 
 from backend.models.characters import Character, CharacterVariant, IgnoredCharacter
+from backend.models.media import Audio, Video
 from backend.resources.base import SiteContentResource
 
 
 class CharacterResource(SiteContentResource):
-    # add related media mapping
+    related_audio = fields.Field(
+        column_name="related_audio",
+        attribute="related_audio",
+        m2m_add=True,
+        widget=ManyToManyWidget(Audio, field="id"),
```

Comment:
I remembered one other thing. It's possible that the export will contain links to media files that were deleted or broken on nuxeo and will not get imported. Could you make sure that this gracefully handles media that doesn't exist, for example by skipping media ids that are not found?